reference/release-notes/1.26.1.md

```diff
@@ -0,0 +1,75 @@
+---
+orphan: true
+---
+# 1.26.1
+
+These release notes cover new features and changes in Anbox Cloud 1.26.1.
+
+Anbox Cloud 1.26.1 is a minor release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
```

Comment:
Anbox Cloud 1.26.1 is a minor release.
->
Anbox Cloud 1.26.1 is a patch release.